Jenkinsfile

```diff
@@ -109,7 +109,7 @@ pipeline {
                     export GPG_TTY=${TTY}
 
                     cd presto
-                    unset MAVEN_CONFIG && ./mvnw -s ${WORKSPACE}/settings.xml -V -B -U -e -T2C deploy \
```

Comment:
this atleast needs to have -T1C or it will be very slow

Reply:
updated

Reply:
updated